rusk-wallet/src/wallet.rs

```diff
@@ -622,6 +622,33 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
 
         Ok(gas_prices)
     }
+
+    /// do a contract call over http
+    pub async fn http_contract_call(
+        &self,
+        contract_id: String,
+        fn_name: &str,
+        fn_args: Vec<u8>,
+    ) -> Result<Option<u32>, Error> {
+        let client = self.state()?.client();
+
+        // query the rusk vm
+        let response = client
+            .call("contracts", Some(contract_id), fn_name, &fn_args)
+            .await?;
+
+        if !response.is_empty() {
+            // wasm can only return u32 right now
```

Comment:
I don't get this, the wasm call return a byte array, not u32 (check how we get the account info)